bin/precice-config-visualizer

```diff
@@ -227,7 +227,7 @@ def parse_args():
 
 def main():
     args = parse_args()
-    xml = parseXML(args.infile.read())
+    xml = parseXMLFile(args.infile.read())
```

Comment:
This sounds like a reasonable change (opening the file, not the content and going from the outer to the inner layer without skipping layers), but I don't understand why it fixes the bug. Do you?

Reply:
No, but [stackoverflow](https://stackoverflow.com/questions/28534460/lxml-etree-xml-valueerror-for-unicode-string) knows,